MaterialLib/PorousMedium/UnsaturatedProperty/CapillaryPressure/VanGenuchtenCapillaryPressureSaturation.h

```diff
@@ -78,6 +86,54 @@ class VanGenuchtenCapillaryPressureSaturation final
 private:
     const double _pb;  ///< Entry pressure.
     const double _m;   ///< Exponent m, m in [0,1]. n=1/(1-m).
+    const bool _has_regularized;
+    const double xi = 1e-5;
```

Comment:
Please explain what xi is, and why it has this particular value.
Also the variables above are documented, but not the two new variables.